### DISCUSSION TITLE
Where to find the OCC ?

### DISCUSSION THREAD

**Discussion by Curtis777** [nextcloud/all-in-one#2719] [Questions] [answered]
Hey Guys, 
i hope im right here... didnt found any helpfull on Google.
or any other site

Where do i find the occ command folder ?
i tried to 
`occ files:scan`
but it says 
`sh: occ: not found`

i guess in the nextcloud-aio-mastercontainer Container but which folder ?

**Comment by Zoey2936** [comment, marked as answer]
please see here: https://github.com/nextcloud/all-in-one#how-to-run-occ-commands
